--- netacct-mysql-0.75/configure.in.orig	Sat Sep 28 18:25:17 2002
+++ netacct-mysql-0.75/configure.in	Mon Mar 10 13:48:27 2003
@@ -126,7 +126,7 @@
   else
 
     for i in mysql lib lib/mysql; do
-      str="$MYSQL_DIR/$i/libmysqlclient.a"
+      str="$MYSQL_DIR/$i/libmysqlclient.so $MYSQL_DIR/$i/libmysqlclient.a"
       for j in `echo $str`; do
         if test -r $j; then
           MYSQL_LIB_DIR="$MYSQL_DIR/$i"
